docs/create-pages.mdx

```diff
@@ -89,6 +89,20 @@ Once this is done, any `<Link />` component or hook from `waku/router` that uses
 
 Note: The file-based router paths will be supported in the future with some form of code-generation to get the types from your local page files.
 
+#### PageProps
+
+The above snippet will also provide types for the `PageProps` type. This will give type safety for the slugs in your pages.
+
+```tsx
+// ./src/pages/about.tsx
+import { PageProps } from 'waku/router/client';
```

Comment:
Importing "client" in the server file isn't very nice.

Hm, I think you meant this?
```suggestion
import type { PageProps } from 'waku/router';
```